src/Microsoft.AspNet.SignalR.Core/Infrastructure/MemoryPoolTextWriter.cs

```diff
@@ -3,10 +3,11 @@
 using System;
 using System.Text;
 using System.IO;
+using System.Globalization;
 
 namespace Microsoft.AspNet.SignalR.Infrastructure
 {
-    public class MemoryPoolTextWriter : TextWriter
+    internal class MemoryPoolTextWriter : TextWriter
```

Comment:
This is what I thought - making this internal saves some FxCop suppressions since we probably don't want to check if the value is not null each time we want to write a chunk...